### PR TITLE
feat(Los Angeles Times): add presence

### PR DESCRIPTION
**Pull request by N0chteil**
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Screenshot_182
Screenshot_184
Screenshot_183
image
Screenshot_190
Screenshot_188
Screenshot_189
</details>

### CODE REVIEW

File: websites/L/Los Angeles Times/presence.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+		};
+
+	if (showButtons && !privacyMode) {
+		presenceData.buttons = [
+			{
+				label: "View Page",
+				url: href,
+			},
+		];
+	}
```
Comment (DarkVillager) [inline websites/L/Los Angeles Times/presence.ts:29]: It's not allowed to show on the home page.

File: websites/L/Los Angeles Times/presence.ts [hunk 2]
```diff
@@ -0,0 +1,202 @@
+const presence = new Presence({
+		clientId: "1005501034191409153",
+	}),
+	time = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/L/Los Angeles Times/presence.ts:4]: Just for repo consistency
```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
```

**Comment by N0chteil** [comment]
Sorry, my iPad's autocorrect

File: websites/L/Los Angeles Times/presence.ts [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+		};
+
+	if (showButtons && !privacyMode) {
+		presenceData.buttons = [
+			{
+				label: "View Page",
+				url: href,
+			},
+		];
+	}
```
Comment (DarkVillager) [inline websites/L/Los Angeles Times/presence.ts:29]: You could remove this check and just place one at the bottom to delete the buttons if the button setting is false?
```suggestion
		presenceData.buttons = [
			{
				label: "View Page",
				url: href,
			},
		];
```

File: websites/L/Los Angeles Times/presence.ts [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+		}
+
+		if (showButtons && !privacyMode) {
+			presenceData.buttons = [
+				{
+					label: "Read Article",
+					url: href,
+				},
+			];
+		}
```
Comment (DarkVillager) [inline websites/L/Los Angeles Times/presence.ts:97]: Same here?
```suggestion
			presenceData.buttons = [
				{
					label: "Read Article",
					url: href,
				},
			];
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/L/Los Angeles Times/presence.ts [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+			default:
+				if (path[1]) {
+					presenceData.details = "Viewing a Category";
+
+					if (!privacyMode) presenceData.state = title;
+				}
+		}
+	}
+
+	if (!presenceData.state) delete presenceData.state;
```
Comment (DarkVillager) [inline websites/L/Los Angeles Times/presence.ts:181]:
```suggestion
	if (!showButtons) delete presenceData.buttons;
	if (!presenceData.state) delete presenceData.state;
```

File: websites/L/Los Angeles Times/presence.ts [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+			presenceData.state = title;
+
+			if (showButtons) {
+				presenceData.buttons = [
+					{
+						label: "View Project",
+						url: href,
+					},
+				];
+			}
```
Comment (DarkVillager) [inline websites/L/Los Angeles Times/presence.ts:116]: Same here
```suggestion
				presenceData.buttons = [
					{
						label: "View Project",
						url: href,
					},
				];
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/L/Los Angeles Times/presence.ts (hunk 3 above)
Comment (EncryptedDev) [reply websites/L/Los Angeles Times/presence.ts:29]: Although it is more repetitive, the former method of doing so is more preformant.

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**